coderd/database/dbfake/dbfake.go

```diff
@@ -1376,13 +1376,26 @@ func (q *FakeQuerier) GetGroupByOrgAndName(_ context.Context, arg database.GetGr
 	return database.Group{}, sql.ErrNoRows
 }
 
-func (q *FakeQuerier) GetGroupMembers(_ context.Context, groupID uuid.UUID) ([]database.User, error) {
+func (q *FakeQuerier) GetGroupMembers(_ context.Context, id uuid.UUID) ([]database.User, error) {
 	q.mutex.RLock()
 	defer q.mutex.RUnlock()
 
+	var isEveryoneGroup bool
+	for _, org := range q.organizations {
+		if org.ID == id {
+			isEveryoneGroup = true
+			break
+		}
+	}
+
+	if isEveryoneGroup {
+		var cp []database.User
+		return append(cp, q.users...), nil
```

Comment:
This will break when we have multiple orgs, right? I'm not sure where all we have assumptions that there is one org, but if it's a broad practice then disregard this comment.

Reply:
It is unfortunately a broad practice. We assume 1 org in quite a few places

Reply:
Well, maybe this PR is the beginning of a new era.